lively.ide/studio/version-checker.cp.js

```diff
@@ -65,8 +69,22 @@ class VersionChecker extends Morph {
     await this.displayLivelyVersionStatus();
   }
 
-  onMouseDown (evt) {
+  async updateLively () {
+    let li;
+    once(L2LClient.default(), 'onReconnect', async () => {
+      li.remove();
+      await $world.inform('Press OK to reload this page and finish the update.');
+      location.reload();
+    });
+    li = $world.showLoadingIndicatorFor($world, 'Updating lively');
+    runCommand('./../update.sh', { l2lClient: ShellClientResource.defaultL2lClient });
```

Comment:
Cant we just bind this to `VersionChecker.cwd()` as well? Here we implicitly assume the default cwd will always be the `lively.server`, and if that will change in the future this code will break out of the blue.